### PR TITLE
Log EOF to info level

### PR DESCRIPTION
**Pull request by Lgsdream**
Signed-off-by: lugang <lugang@zhuanzhuan.com>

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -78,8 +85,14 @@ func (c *client) serve(conn net.Conn) error {
 		default:
 			cmd, err = c.readCommand()
 			if err != nil {
-				zap.L().Error("read command failed", zap.String("addr", c.cliCtx.RemoteAddr),
-					zap.Int64("clientid", c.cliCtx.ID), zap.Error(err))
+				if err == io.EOF {
+					zap.L().Info("close connection", zap.String("addr", c.cliCtx.RemoteAddr),
+						zap.Int64("clientid", c.cliCtx.ID))
+					err = nil
```
Comment (shafreeck) [inline client.go]: I suggest to just return here. Maybe you could put `c.conn.Close()` somewhere before this line

Comment (Lgsdream) [reply client.go]: OK

File: client.go [hunk 2]
```diff
@@ -53,13 +54,19 @@ func (c *client) Write(p []byte) (int, error) {
...
-			zap.Int64("clientid", c.cliCtx.ID),
-			zap.String("namespace", c.cliCtx.Namespace),
-			zap.Bool("multi", c.cliCtx.Multi),
-			zap.Bool("watching", c.cliCtx.Txn != nil),
-			zap.String("command", c.cliCtx.LastCmd),
-			zap.String("error", err.Error()))
+		if err == io.EOF {
+			zap.L().Info("close connection", zap.String("addr", c.cliCtx.RemoteAddr),
+				zap.Int64("clientid", c.cliCtx.ID))
+			err = nil
```
Comment (shafreeck) [inline client.go]: It is weird to set err to nil here. Maybe we can do this:

```
if err != nil {
  if err == io.EOF {
    log.Info...
  } else {
    log.Error...
    return 0, err
  }
}
return n, nil
```

Comment (Lgsdream) [reply client.go]: Ok， I've changed it as you said

**Pull request merged**